pip_audit/_cache.py

```diff
@@ -152,6 +152,13 @@ def caching_session(cache_dir: Optional[Path], *, use_pip: bool = False) -> Cach
     directory, falling back on the internal `pip-audit` cache directory if the user's
     version of `pip` is too old.
     """
+
+    # We limit the number of redirects to 5, since the services we connect to
+    # should really never redirect more than once or twice.
+    inner_session = requests.Session()
+    inner_session.max_redirects = 5
```

Comment:
Flagging for review: this limit isn't strictly necessary, but the default (30) seems excessive in our case and lowering it will improve our responsiveness.